src/init.cpp

```diff
@@ -808,7 +808,13 @@ void InitLogging()
     fLogIPs = gArgs.GetBoolArg("-logips", DEFAULT_LOGIPS);
 
     LogPrintf("\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n");
-    LogPrintf("Bitcoin version %s\n", FormatFullVersion());
+    std::string version_string = FormatFullVersion();
+#ifdef DEBUG
+    version_string += " (debug build)";
+#else
+    version_string += " (release build)";
+#endif
+    LogPrintf("Bitcoin version %s\n", version_string);
```

Comment:
Should say "Bitcoin Core" according to the rebranding 4 years ago: #3408

Reply:
I didn't introduce this message here, but sure, can update this.

Reply:
I've changed it to use PACKAGE_NAME (which is Bitcoin Core), same as for the `--help` messages.